transfermethodrepository/src/test/java/com/hyperwallet/android/ui/transfermethod/repository/TransferMethodRepositoryImplTest.java

```diff
@@ -157,6 +159,23 @@ public Object answer(InvocationOnMock invocation) {
         assertThat(mErrorsArgumentCaptor.getValue().getErrors(), hasItem(error));
     }
 
+    @Test
+    public void testCreateTransferMethod_createTransferWithUnsupportedTransferType() {
```

Comment:
I think testCreateTransferMethod_withUnsupportedTransferMethodType would be better